draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -16,21 +17,53 @@ const getRelativeParent = (element) => {
 };
 
 export default class Toolbar extends React.Component {
-
-  state = {
-    isVisible: false,
-    position: undefined,
-
+  static propTypes = {
     /**
-     * If this is set, the toolbar will render this instead of the regular
-     * structure and will also be shown when the editor loses focus.
-     * @type {Component}
+     * An indents from the left / right window borders
      */
-    overrideContent: undefined
+    toolbarMargin: PropTypes.number
```

Comment:
why can we not just use css for this?

Reply:
Hi, I did some research today. This prop is not really related with css, it is used as some abstract argument while positioning. So, if we will use css margin instead of this prop, that will not give us any advantages, except for increasing calculation complexity, because will then will have to take into account these css margins.

Reply:
So, if you insist, we can implement css margins